src/config.h

```diff
@@ -391,10 +391,4 @@ void setcpuaffinity(const char *cpulist);
 #define ATTRIBUTE_TARGET_AVX512
 #endif
 
-#if defined(__linux__) && (defined(__GNUC__) && (__GNUC__ > 4) || defined(__clang__) && (__clang_major__) > 5)
```

Comment:
Can't we just add `defined(__GLIBC__)` to this check? Like we did here:

* #1707

One more thing: Does ifunc require linux or just glibc? It's possible to run freebsd with glibc. There's a debian variant with this combo.

Reply:
The ifunc feature requires ELF. Even though BSD systems have glibc, I'm not sure if there are any gaps between the loader and runtime linker compared to Linux.